test/testcafe/spec/IdentifyWithPassword_spec.js

```diff
@@ -139,7 +138,6 @@ test.meta('v3', false).requestHooks(identifyRequestLogger, identifyWithPasswordM
   await t.expect(await identityPage.hasShowTogglePasswordIcon()).notOk();
 });
 
-// OKTA-555502 - showPasswordToggleOnSignInPage feature not supported yet
 test.meta('v3', false).requestHooks(identifyRequestLogger, identifyWithPasswordMock)('should not have password toggle if "features.showPasswordToggleOnSignInPage" is false', async t => {
```

Comment:
this looks like a dupe of the previous test.  Could you change this to test the default case where `features.showPasswordToggleOnSignInPage` is undefined?

Reply:
This is the test that tests the "dot notation" way of passing the FF which is not supported in gen3. We need to assess if we want to support this (it's not documented) but for the moment we can just skip the test in gen3.

@trevoring-okta can you add a comment with a link to the new Jira and also a comment explaining why we skip it right now?

Reply:
Ohh I see okay got it my mistake.  Adding the comment will help distinguish each test